BLSQ/iaso
# WC2-591: Integrate Account into beneficiary table

Explain what problem this PR is resolving

Related JIRA tickets : [WC2-591](https://bluesquare.atlassian.net/browse/WC2-591)

## Self proofreading checklist



## Changes

As there is data linked to 2 account, adding a new field into beneficiary table is required in order to separate the data linked to 2 country(South Sudan and Nigeria). 


## How to test

From the local instance with wfp coda database, run following tasks:

- creating a new migration for adding new field `account` into `wfp_beneficiary` table: `docker compose run iaso manage makemigrations wfp`
- adding the new field into `wfp_beneficiary` table:  `docker compose run iaso manage migrate wfp`
- generating data for tableau dashboard : `docker compose run iaso manage wfp_etl`

Then login on django admin and check on [beneficiary table](http://localhost:8081/admin/wfp/beneficiary/) . You should see the account field.

## Print screen / video

[Bneeficiary-Administration-de-Iaso-Iaso.webm](https://github.com/user-attachments/assets/7aac3300-8103-4d08-8537-4d87dab6ba4e)



[WC2-591]: https://bluesquare.atlassian.net/browse/WC2-591?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
